lib/classes/Utils.js

```diff
@@ -332,6 +287,12 @@ class Utils {
     return [];
   }
 
+  validateProviders(provider) {
```

Comment:
What do we need this function for? I don't think we should limit this in code, because people should be able to add more providers